app/models/roster_entry.rb

```diff
@@ -10,19 +10,23 @@ class IdentifierCreationError < StandardError; end
 
   before_create :validate_identifiers_are_unique_to_roster
 
-  def self.to_csv
+  # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+  def self.to_csv(user_to_group_map = {})
     CSV.generate(headers: true, col_sep: ",", force_quotes: true) do |csv|
-      csv << %i[identifier github_username github_id name]
+      csv << %i[identifier github_username github_id name group_name]
```

Comment:
I feel we should only include the `group_name` column if `user_to_group_map` is not empty.
Alternatively this could be written as follows:
```ruby
columns = %i[identifier github_username github_id name]
columns << :group_name unless user_to_group_map.empty?
csv << columns
```